hexa/pipelines/tests/test_schema/test_pipelines.py

```diff
@@ -1471,6 +1471,55 @@ def test_pipelines_permissions_schedule_required_param_with_no_default_value(sel
             r["data"]["pipelineByCode"],
         )
 
+    def test_pipelines_permissions_schedule_required_param_with_no_default_value_with_version_config(
```

Comment:
Could you write more tests to check how the config is merged when you run a pipeline with a version's config?